node/flags/flags.go

```diff
@@ -294,45 +294,6 @@ var (
 		Value:    units.GiB, // intentionally large for the time being
 	}
 
-	// Test only, DO NOT USE the following flags in production
```

Comment:
This is a drop-by fix, as we happen to need the `OverrideStoreDurationBlocksFlag` below.

`DO NOT USE the following flags` is not correct as other production flags were added it below

Fixed it, and made it even more clear about the test flag section